### PR TITLE
[SM-877] Updating trash to have ability to copy UUID

### PR DESCRIPTION
**Pull request by cd-bitwarden**
## Type of change

```
- [ x ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add ability to copy UUID from Trash page for secret.
## Code changes

**bitwarden_license/bit-web/src/app/secrets-manager/trash/trash.component.html** - Set up copy event
**bitwarden_license/bit-web/src/app/secrets-manager/trash/trash.component.ts** - Adding services and call to logic for copying UUID
## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by nsbgit — APPROVED** [review]
LGTM

**Review by Thomas-Avery — CHANGES_REQUESTED** [review]
The layout of this in the trash section doesn't look like the other secret lists and the FIGMA.

Looks like we need to adjust the ordering of some `<div>`s related to if the name appears for trash or not inside `secrets-list.component.html`

image

vs 

image

**Comment by cd-bitwarden** [comment]
> The layout of this in the trash section doesn't look like the other secret lists and the FIGMA.
> 
> Looks like we need to adjust the ordering of some `<div>`s related to if the name appears for trash or not inside `secrets-list.component.html`
> 
> image
> 
> vs
> 
> image

Thank you for catching this 😺

File: bitwarden_license/bit-web/src/app/secrets-manager/shared/secrets-list.component.html [hunk 1]
```diff
@@ -75,6 +75,7 @@
                 {{ secret.name }}
               </button>
             </div>
+            <div *ngIf="trash">{{ secret.name }}</div>
```
Comment (Thomas-Avery) [inline bitwarden_license/bit-web/src/app/secrets-manager/shared/secrets-list.component.html:78]: If I'm understanding this correctly, when `trash` is true, we will get an extra div? 

```
            <div>
                // button doesn't display but div is still included
            </div>
            <div *ngIf="trash">{{ secret.name }}</div>
```

What do you think about either moving the `*ngIf="!trash"` to the div or something like an else block? https://angular.io/api/common/NgIf#showing-an-alternative-template-using-else

**Review by Thomas-Avery — CHANGES_REQUESTED** [review]

**Review by Thomas-Avery — APPROVED** [review]
Looks good, thank you!

**Comment by cd-bitwarden** [comment]
@Thomas-Avery there were some merge conflicts I had to resolve, can you please give one last review? 😄 Thank you!

**Review by Thomas-Avery — APPROVED** [review]
LGTM

**Pull request merged**